.github/workflows/build-test-package.yml

```diff
@@ -13,17 +13,17 @@ jobs:
           - os: ubuntu-18.04
             c-compiler: "gcc"
             cxx-compiler: "g++"
-            itk-git-tag: "v5.1.2"
+            itk-git-tag: "abd38d5a0040b9a8fbb0ad3127089dbb72a93342"
```

Comment:
Should this itk-git-tag also be "v5.2rc01", rather than "abd38..."?

Reply:
No, it should be abd38d5. There were some fixes added after RC1 was tagged.

Reply:
@dzenanz Thanks, so could you say which fixes were essential for the ITKElastix CI build? Maybe it's time for RC2... 😸

Reply:
Yes, RC2 is planned 😄

Meanwhile we should merge this.